src/quacc/utils/files.py

```diff
@@ -61,15 +61,15 @@ def copy_decompress(source_files: list[str], destination: str) -> None:
     """
     for f in source_files:
         z_path = zpath(f)
-        if os.path.exists(z_path):
+        if Path.exists(Path(z_path)):
             z_file = os.path.basename(z_path)
-            copy(z_path, os.path.join(destination, z_file))
-            decompress_file(os.path.join(destination, z_file))
+            copy(z_path, Path(destination, z_file))
+            decompress_file(Path(destination, z_file).as_posix())
```

Comment:
I'm a little hesitant about calling `.as_posix()` since we also want to support Windows. However, it looks like the Windows tests passed so I guess it didn't do anything terrible. Any idea why this is needed?